src/data/injections.js

```diff
@@ -123,15 +123,15 @@ const AVAILABLE_INJECTIONS = [
   },
   {
     id: "bug1561371",
-    platform: "android",
-    domain: "mail.google.com",
+    platform: "desktop",
+    domain: "cloud.google.com",
     bug: "1561371",
```

Comment:
We might want to update this to whatever the right bug # is for the cloud issue, rather than leaving it pointing to the gmail issue.